lib/xapi/implementation.rb

```diff
@@ -91,6 +91,8 @@ def assemble_readme
 #{readme_body}
 
 #{problem.source_markdown}
+
+#{incomplete_solutions_body}
```

Comment:
Do we have other text about submitting a solution? If so, where does it come from?